sentry/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG SENTRY_IMAGE
+FROM ${SENTRY_IMAGE}
+
+COPY . /usr/src/sentry
+
+RUN if [ -s /usr/src/sentry/enhance-image.sh ]; then \
+    /usr/src/sentry/enhance-image.sh; \
+fi
+
+RUN if [ -s /usr/src/sentry/requirements.txt ]; then \
+    echo "sentry/requirements.txt is deprecated, use sentry/enhance-image.sh - see https://github.com/getsentry/self-hosted#enhance-sentry-image"; \
+    pip install -r /usr/src/sentry/requirements.txt; \
```

Comment:
This covers the functionality removed from `entrypoint.sh`, yes? The checksum shenanigans are not necessary to bring forward?

Reply:
As far as I can tell, they were only added with https://github.com/getsentry/self-hosted/pull/834 to maintain some level of caching. Since the image build is naturally run only once, it is no longer necessary to cache.

Reply:
Yup, exactly as @spawnia said. I just poorly replicated what Docker layered file system automatically does.